.github/workflows/stronghold.yml

```diff
@@ -0,0 +1,20 @@
+name: stronghold
+
+on:
+  pull_request:
+    paths: 'tools/stronghold/**'
```

Comment:
Let's include the path of this workflow in here too?

Reply:
good idea!